src/librbd/AioObjectRequest.h

```diff
@@ -93,6 +96,7 @@ class AioObjectRequest : public AioObjectRequestHandle {
   Context *m_completion;
   Extents m_parent_extents;
   bool m_hide_enoent;
+  const blkin_trace_info *m_trace_info;
```

Comment:
I wondered why ZTracer::Trace wasn't being used but I was told to pass the trace info to send_write_op. I will try to change it, can you please explain more about the issue with the trace info pointers? What problems can it cause?